lua/lspsaga/lspkind.lua

```diff
@@ -56,27 +56,27 @@ local function find_index_by_type(k)
   return nil
 end
 
-local function gen_symbol_winbar_hi()
-  local prefix = 'LspSagaWinbar'
-  local winbar_sep = 'LspSagaWinbarSep'
-
-  if next(custom_kind) ~= nil then
-    for k, conf in pairs(custom_kind) do
-      local index = find_index_by_type(k)
-      if not index then
-        vim.notify('Does not find this type in kind')
-      end
+if next(custom_kind) ~= nil then
```

Comment:
you made if outside . so this statement will be effect when lspkind.lua require.  should be set into a init kind funciton. then require it in saga init.

```lua
function load_custom_kind()
  if next(custom_kind) == nil then
     return
  end
 do some stuff
end

```

then in init_saga function require this function .

Reply:
Fixed :)